src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -43,277 +48,1082 @@ public static void ClassCleanup()
+            page.CalculatorApp.EnsureCalculatorHasFocus();
+            page.NavigationMenu.ChangeCalculatorMode(CalculatorMode.ProgrammerCalculator);
+            Assert.AreEqual("Programmer", page.CalculatorApp.GetCalculatorHeaderText());
+            page.StandardAoTCalculatorPage.GetAoTPresence();
+            Assert.AreEqual("False", page.StandardAoTCalculatorPage.GetAoTPresence());
+
+            page.CalculatorApp.EnsureCalculatorHasFocus();
+            page.NavigationMenu.ChangeCalculatorMode(CalculatorMode.DateCalculator);
+            Assert.AreEqual("Date Calculation", page.CalculatorApp.GetCalculatorHeaderText());
+            page.StandardAoTCalculatorPage.GetAoTPresence();
+            Assert.AreEqual("False", page.StandardAoTCalculatorPage.GetAoTPresence());
+
+            page.CalculatorApp.EnsureCalculatorHasFocus();
+            page.NavigationMenu.ChangeCalculatorMode(CalculatorMode.StandardCalculator);
+            Assert.AreEqual("Standard", page.CalculatorApp.GetCalculatorHeaderText());
+            page.StandardAoTCalculatorPage.GetAoTPresence();
+            Assert.AreEqual("True", page.StandardAoTCalculatorPage.GetAoTPresence());
         }
 
         [TestMethod]
-        public void Memory_SubtractTest()
+        [Priority(2)]
+        public void AoT_Scaling()
         {
-            page.StandardOperators.NumberPad.Input(12);
-            page.StandardOperators.DivideButton.Click();
-            page.StandardOperators.NumberPad.Input(3);
-            page.StandardOperators.EqualButton.Click();
-
-            page.MemoryPanel.MemButton.Click();
-            page.StandardOperators.NumberPad.Input(3.3);
-            page.MemoryPanel.MemMinus.Click();
-            var memoryItems = page.MemoryPanel.GetAllMemoryListViewItems();
-            Assert.IsTrue(memoryItems[0].Text.Equals("0.7", StringComparison.InvariantCultureIgnoreCase));
+            page.StandardAoTCalculatorPage.NavigateToStandardAoTMode();
+            page.StandardAoTCalculatorPage.AoTWindowSizeWithinRange();
+            Size smallAoTWindowSize = new Size(161, 168);
+            Size largeAoTWindowSize = new Size(502, 502);
+            Size mediumAoTWindowSize = new Size(396, 322);
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = smallAoTWindowSize;
+            Assert.AreEqual("(161, 168)", page.CalculatorApp.GetCalculatorWindowSize());
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = largeAoTWindowSize;
+            Assert.AreEqual("(502, 502)", page.CalculatorApp.GetCalculatorWindowSize());
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = mediumAoTWindowSize;
+            Assert.AreEqual("(396, 322)", page.CalculatorApp.GetCalculatorWindowSize());
+            page.StandardAoTCalculatorPage.NavigateToStandardMode();
         }
 
         [TestMethod]
-        public void Memory_RecallTest()
+        [Priority(2)]
+        public void AoT_ScaleRetention()
```

Comment:
Do we need this test? Are we doing special logic in AoT that would affect the scale retention of the app? If not, then we are likely just testing Compact Overlay and UWP.

Reply:
This automated test was created to reduce manual testing.  Whether this Compact Overlay and UWP or specific to calculator, it does need to be tested to ensure that it does not break.  We can remove it from automation, but it should not be removed from testing.